src/main/java/com/google/finapp/ArgsModule.java

```diff
@@ -91,6 +98,9 @@ String provideSpannerDatabaseId() {
 
     @Parameter(names = {"--spanner_database_id"})
     String spannerDatabaseId;
+
+    @Parameter(names = {"--spanner_use_jdbc"}, arity = 0)
```

Comment:
Optional: If we want `arity = 0`, we can safely remove it,  this is the default.